index.html

```diff
@@ -10,219 +10,268 @@
+    
+                    <ul class="nav-bar-ul">
+                        <a href="#"><li>Portoflio</li></a>
+                        <a href="#"><li>About</li></a>
+                        <a href="#"><li>Contact</li></a>
+                      </ul>
+                </nav>
+            </header>
+            <div class="headline">
+                <h1>I’m Jonathan<br> Glad to see you!</h1>
+                
+                <p class="text">
+                    I’m a software developer! I can help you build a product , feature or
+                    website Look through some of my work and experience! If you like what
+                    you see and have a project you need coded, don’t hestiate to contact
+                    me.
+                </p>
+                <h2 class="hex connect">LET’S CONNECT</h2>
+                <ul class="social-media">
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/twitter.svg" /></a>
+                    </li>
+                    <li clalss="socials">
+                        <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+                    </li>
+                    <li clalss="socials">
+                        <a href="#"><img src="images/socials/m.svg" /></a>
+                    </li>
+                    <li clalss="socials">
+                        <a href="#"><img src="images/socials/Vector1.svg" /></a>
+                    </li>
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/Vector2.svg" /></a>
+                    </li>
+                </ul>
+            </div>
+    </section>   
+        <div class="grid-items other-projects">
+            <div>
+                <img class="project-img" src="images/projects/project-1.png" alt="Project-1" />
+            </div>
+            <div>
+                <div class="details">
+                    <h3 class="project-title">Tonic</h3>
+                    <ul class="canopy">
+                        <li>CANOPY</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">Back End Dev</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">2015</li>
+                     </ul>
+                </div>
+                <div>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+
+                </div>
+                <div>
+                    <ul class="categories hex">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                </div>
+                <div class="see-project">
+                    <button type="button" class="hex">See Project</button>
```

Comment:
- [ ] Kindly note that for the button hover effect, the color of the text is also supposed to change to a white-like color to enable a pixel perfect design 🙃.